Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -620,11 +627,25 @@ - (BranchLinkProperties *)getFirstReferringBranchLinkProperties {
 }
 
 - (NSDictionary *)getFirstReferringParams {
-    return [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
+    NSDictionary *origInstallParams = [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
```

Comment:
let's move this to after the `if` statement. there's no reason to execute this code if `self.deepLinkDebugParams` is true